src/main/java/jdbc/web/MainServlet.java

```diff
@@ -52,13 +60,24 @@ private void sendResponse(ViewModel vm, HttpServletRequest req, HttpServletRespo
             res.addCookie(cookie);
         }
 
-        if (req.getMethod().equals("POST") && path.equals("/servlet/login")) {
+        if (req.getMethod().equals("POST") && path.equals("/servlet/login") && vm.getView().equals("home")) {
             req.getSession().setAttribute("LOGGED_USER", new User());
+            res.sendRedirect("/Lab8_war_exploded/servlet/home");
+        } else if (req.getMethod().equals("POST") && path.equals("/servlet/register")) {
+            res.sendRedirect("/Lab8_war_exploded/servlet/login");
+        } else if (req.getMethod().equals("GET")
```

Comment:
@Dimitriy1 to complex boolean statement. Move to separate method. See: https://simpleprogrammer.com/refactoring-boolean-conditions-into-methods/
Also, if I am visiting **path.equals("/servlet/home")** should I be redirected to **"/Lab8_war_exploded/servlet/login"** ?

Reply:
@boroda4436 If I am logged in and I want visit home - I visit home, but If I am not logged in I can't visit home,